src/index.html

```diff
@@ -3,12 +3,51 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <title>Moyo</title>
+    <link href="https: //fonts.googleapis.com/css2? family= Montserrat:wght@400;500;600 & family= Roboto:ital,wght@1,300;1,400;1,500 & display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+        src="images/logo.png"
+        alt="logo image"
+        class="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple.html" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung.html" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones.html" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#laptops.html"
+            class="nav__link"
+            data-qa="hover">Laptops & Computers</a>
```

Comment:
```suggestion
            <a 
              href="#laptops.html"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```
![image](https://user-images.githubusercontent.com/96409563/236263622-a9c3d65f-517c-4835-a005-74dda5fb68d2.png)